src/expandable-section/expandable-section-header.tsx

```diff
@@ -95,14 +95,15 @@ const ExpandableNavigationHeader = ({
   icon,
 }: ExpandableNavigationHeaderProps) => {
   return (
```

Comment:
`<ExpandableNavigationHeader />` is rendered conditionally only for the `navigation` variant, so the change below is only applied to this variant.

https://github.com/cloudscape-design/components/blob/4c020a82878a4590218caacad2468144e033795a/src/expandable-section/expandable-section-header.tsx#L255-L265